### ISSUE TITLE
require() with backticks (string template) causing `Internal error: Not_found`

### ISSUE THREAD

**Issue by turadg** [facebook/flow#4041]
This worked in 0.45 and fails in 0.47,
```
      return <Image src={require(`assets/images/foo.png`)} />;
```

Single quotes works:
```
      return <Image src={require('assets/images/foo.png')} />;
```
And templates that use params works:
```
      return <Image src={require(`assets/images/${param}.png`)} />;
```

I thought it was something to do with superfluous string templates, but even this simple [<Image src={require('image.png')} />](https://flow.org/try/#0DwSQtghg5gpgBAZwE4GMC8BvJMCOBXAS2wAoByAyWAOgAcA7KUgSgF84B6APiA) outputs internal error.

**Comment by rivertam** [comment]
I'm getting the same thing, but I'm not even trying to require an asset or anything fancy. I'm just trying the file

```javascript
/* @flow */
const { pick } = require(`lodash/fp`)
```

Seems like... a pretty major bug, but maybe I'm doing something wrong?

**Comment by rivertam** [comment]
Ah, so I can only reproduce if I'm using backticks. Minimal reproduction repository: https://github.com/rivertam/flow-bug

**Comment by hackhat** [comment]
Any solution found?

**Comment by dashed** [comment]
Can confirm that backticks as in comment https://github.com/facebook/flow/issues/4041#issuecomment-304468239 can cause issues:

```
require(`redux-logger`);
```

Error output:

```
Error: src/store/index.js:0
Internal error: Context.Require_not_found("redux-logger")
```

**Comment by turadg** [comment]
I've clarified the issue description. Also this is no longer a problem in latest Flow so I'm closing. (I don't know when it was fixed but I'm running 0.60.1 and what I reported works now.)
